NeuralNet.js

```diff
@@ -0,0 +1,171 @@
+class NeuralNet {
+    constructor(options) {
+        this.config = {
+            inputs: options.inputs || 3,
+            outputs: options.outputs || 2,
+            debug: options.debug || true,
+            outputLabels: options.outputLabels || [],
+            inputLabels: options.inputLabels || [],
+            batchSize:options.batchSize || 64,
+            epochs: options.epochs || 32,
+            NO_VAL: null
+        }
+        this.config.NO_VAL = this.config.outputs;
+        this.model = null;
+    }
+
+
+    async createModel() {
+
+        this.model = tf.sequential();
+
+        const hidden = tf.layers.dense({
+            units: 16,
+            inputShape: [this.config.inputs],
+            activation: 'relu'
+        });
+        
+        const output = tf.layers.dense({
+            units: this.config.outputs,
+            activation: 'softmax'
```

Comment:
`softmax` wouldn't be the default, but only if specified as a "classification" task.

Reply:
Oh! yes, maybe something then like:

```
new ml5.neuralNetwork(3, 2, 'regression') 
new ml5.neuralNetwork(3, 2, 'classification') 
```

then our activation function will change accordingly

Reply:
Yes! And oddly enough I think I would default to regression, having the user specify only classification. Like a vanilla neural network natively is just going to give numbers, not output labels. (Most of these comments are about me thinking ahead to my interactive training a la wekinator examples).

Reply:
Yes, I also think this makes sense. I also think about the Nature of Code behaviors which also operate this way.

Reply:
Indeed! I am going to work on revising Chapter 10 this week and have it as reading for the course next week!

Reply:
Awesome! Ok, I will ping you as I make updates on the neuralnet class!